_posts/wookiemage/2016-02-24-Colins-Final-Drawing-App.md

```diff
@@ -0,0 +1,36 @@
+---
+author: wookiemage
+layout: post
+title: "Colin's Drawing App
+"
```

Comment:
automation is awesome but can amplify small mistakes :)

On Thu, Feb 25, 2016 at 10:38 AM wookiemage <notifications@github.com>
wrote:

> In _posts/wookiemage/2016-02-24-Colins-Final-Drawing-App.md
> <https://github.com/silshack/spring2016/pull/661#discussion_r54113325>:
>
> > @@ -0,0 +1,36 @@
> > +---
> > +author: wookiemage
> > +layout: post
> > +title: "Colin's Drawing App
> > +"
>
> Seems like it didn't do anything. That's what I get for using Phrase
> Express to automate post creation...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/silshack/spring2016/pull/661/files#r54113325>.
>